Rethinking_2/Chp_02.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>I think this should be <code>ax[idx].set_title(f'{ps} points')</code>, otherwise the title is not informative.</p><p>Also, I think we should use f-strings in the whole repo whenever possible, instead of the <code>.format</code> formulation -- it's faster, easier to read and was introduced in python 3.6 so people had time to adjust now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>